package.json

```diff
@@ -56,26 +57,25 @@
   "engines": {
     "node": ">=14.16.0"
   },
-  "dependencies": {
-    "resolvewithplus": "^2.0.0"
-  },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.38.1",
+    "@typescript-eslint/parser": "^5.38.1",
     "c8": "^7.12.0",
+    "esbuild": "^0.15.12",
     "eslint": "^8.23.1",
     "eslint-plugin-markdown": "^3.0.0",
-    "@typescript-eslint/parser": "^5.38.1",
-    "@typescript-eslint/eslint-plugin": "^5.38.1"
+    "resolvewithplus": "^2.0.0"
   },
   "scripts": {
+    "build": "esbuild src/esmock.js --minify --bundle --platform=node --format=esm --outfile=./dist/esmock.js",
     "test:install": "cd tests && npm run install:all",
     "test:all": "cd tests && npm run test:all",
     "test:all-ci": "cd tests && npm run test:all-ci",
-    "test": "npm run test:all",
-    "test-ci": "npm run test:install && npm run test:all-ci",
+    "test": "npm run build && npm run test:all",
+    "test-ci": "npm run build && npm run test:install && npm run test:all-ci",
     "test-cover": "npm run test:install && c8 npm run test:all",
     "lint": "eslint --ext=.js,.mjs .",
     "lint-fix": "eslint --ext=.js,.mjs --fix .",
-    "mini:pkg": "npm pkg delete scripts devDependencies dependencies",
```

Comment:
I've moved `resolvewithplus` to devdeps, so users won't install it, and  `mini:pkg` is no longer needed.